b2g_util/get_crashreports.py

```diff
@@ -59,12 +60,20 @@ def get_crashreports(self, serial=None):
         print('Submitted Crash Reports:\n{}\n'.format(submitted))
 
         if retcode_submitted == 0:
+            submitted_url_list = []
             print('The links of Submitted Crash Reports:')
             for line in submitted.split('\n'):
                 submmited_id = re.sub(r'\.txt\s*$', '', re.sub(r'^.+bp-', '', line))
                 submitted_url = 'https://crash-stats.mozilla.com/report/index/{}'.format(submmited_id)
+                submitted_url_list.append(submitted_url)
                 print(submitted_url)
 
+        if self.args.log_json is not None:
+            with open(self.args.log_json, 'w') as outfile:
+                result = {'PendingCrashReportsList': pending, 'SubmittedCrashReportsList': submitted,
```

Comment:
`pending` and `submitted` are the stdout of adb shell command, not list object.
I will suggest to remove them from result object.